jenkins/edx_platform_test_notifier.py

```diff
@@ -94,6 +94,28 @@ def notify_tests_completed_marker(self, pr):
         else:
             return True
 
+    def generate_notification_message(self, pr):
+        failing_contexts = self.get_failures(pr)
+        if not failing_contexts:
+            status_description = "There were no failures."
+        else:
+            status_description = "The following contexts failed:\n{}".format(
+                '\n'.join(["* {}".format(f) for f in failing_contexts])
+            )
+        comment = "Your PR has finished running tests. {}".format(
+            status_description
+        )
+        return comment
+
+    def get_failures(self, pr):
+        """ return a list of the contexts that recently failed on a given pr """
+        head = pr.get_commits().reversed[0]
+        failures = filter(
+            lambda status: status.state == "failure",
```

Comment:
It looks like there's an "error" status also, should we include occurrences of that? https://developer.github.com/v3/repos/statuses/